### PR TITLE
bulk move all other presubmit jobs to config/jobs

### PR DESCRIPTION
**Pull request by krzyzacy**
ref https://github.com/kubernetes/test-infra/issues/8485

left some periodics to go :-)
/assign @BenTheElder

### CODE REVIEW

**Comment by krzyzacy** [comment]
cc|fyi|spam (pulled the list from repo owners, and you are now part of the new subdir owner for your repo as well.) @nikhiljindal @ICGog @ms705 @shivramsrivastava @m1093782566 @apelisse @hoegaarden @totherme @font @madhusudancs @perotinus @pmorie @calebamiles @castrojo @cblecker @grodrigues3 @idvoretskyi @jdumars @parispittman @DirectXMan12 @piosz @x13n @kawych @loburm @mwieglus @huangyuqi @andyxning

File: jobs/config_test.py [hunk 1]
```diff
@@ -143,13 +143,9 @@ def load_prow_yaml(self, path):
         for item in doc.get('periodics'):
             self.add_prow_job(item)
 
-        if 'postsubmits' not in doc:
-            self.fail('No postsubmits in prow config!')
-
         self.presubmits = doc.get('presubmits')
-        postsubmits = doc.get('postsubmits')
```
Comment (krzyzacy) [inline jobs/config_test.py]: I probably just gonna ditch this file entirely :-)

**Comment by m1093782566** [comment]
Got it!

Thanks @krzyzacy

**Comment by krzyzacy** [comment]
/hold cancel
:shipit: :eyes:

**Pull request merged**